src/models/Instructor.js

```diff
@@ -6,11 +6,6 @@ const instructorSchema = new mongoose.Schema(
       first: String,
       last: String,
     },
-    user: {
```

Comment:
As far as I know, we should be storing a reference to the user on the instructors still, right?

Reply:
We are storing a reference to the instructor on the user now. This was a suggestion by Justin.